Lib/test/datetimetester.py

```diff
@@ -61,7 +61,7 @@ def test_constants(self):
         self.assertEqual(datetime.MAXYEAR, 9999)
 
     def test_name_cleanup(self):
-        if '_Fast' not in str(self):
+        if self._test_type == 'Pure':
             return
```

Comment:
Please replace `return` with `self.skipTest(...)`. It is better if the skipped test reported as skipped rather than passed.